configextractor.py

```diff
@@ -10,6 +11,54 @@
 cl_engine = forge.get_classification()
 
 HEURISTICS_MAP = {"malware": 1, "safe": 2}
+# This dict contains fields that we care about, and the corresponding tag if they exist
+FIELD_TAG_MAP = {
+    # TODO: sometimes a domain is returned for address, which should be tagged as network.dynamic.domain
+    # TODO: sometimes an IP:port is returned for address, which should be split into IP and tagged as network.dynamic.ip and port and tagged as network.port
```

Comment:
this is something I discussed with @cccs-jp a while ago. splitting the domains into ip and port is possible but we would lose information in the process. An analyst would be unable to tell which ip is associated with which port.

Reply:
Correct!

Reply:
Alright makes sense, would it be beneficial to have `some_ip:some_port`, tag it as `network.dynamic.uri`, then tag `some_ip` as `network.dynamic.ip` and `some_port` as `network.port`?

Reply:
I don't think so, all addresses already are tagged with network.dynamic.uri and ports are tagged now as well. From the user's persepctive they can already see it's an ip address and an associated port, wouldn't add much value.

Reply:
From discussion earlier today, the decision is to combine `some_domain` or `some_ip` with the corresponding `some_port` and then tag that as `network.dynamic.ip`